src/node_crypto.cc

```diff
@@ -1974,7 +1974,78 @@ static Local<Object> X509ToObject(Environment* env, X509* cert) {
 }
 
 
-// TODO(indutny): Split it into multiple smaller functions
+// Put issuer inside the object
+static void AddIssuer(X509** cert,
+                      const STACK_OF(X509)* const peer_certs,
+                      Local<Object>* const info,
```

Comment:
You don't have to pass a pointer since you don't replace the Object, just add properties to it.

Reply:
Thanks, I've changed this now (locally).